code/modules/antagonists/changeling/powers/_biodegrade_bioacid.dm

```diff
@@ -0,0 +1,26 @@
+/datum/reagent/toxin/acid/bio_acid
+	name = "adaptive bio-acid"
+	description = "An immensely strong, acidic substance of seemingly biological origin. It is teeming with microscopic\
+	organisms that seem to alter its composition to most adaptively dissolve whatever it comes into contact with."
+	color = "#9455ff"
+	creation_purity = 100
+	toxpwr = 0
+	acidpwr = 0
+	ph = 0.0
+	penetrates_skin = TOUCH
+
+/datum/reagent/toxin/acid/bio_acid/expose_mob(mob/living/exposed_mob, methods = TOUCH, reac_volume, show_message = TRUE, touch_protection)
+	. = ..()
+	if(IS_CHANGELING(exposed_mob))
+		to_chat(exposed_mob, span_changeling("We excrete a bio-agent to neutralize the bio-acid. It is routine and reflexive to do so."))
```

Comment:
is there any cue besides this like on the main screen that signals they tried to acid you?

Reply:
Sorry I was asking was a non-textbox equivalent of this message like a balloon alert or something, I guess the funky span makes it pop out so its obvious to the person.

(context being I was wondering if this needed a balloon alert or something for the acid receiver)